config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  get "homepages/index"
+  root to: "homepages#index" # where do i want this to be
+  resources :users, only: [:show, :index]
+  resources :works do
+    resources :votes, only: [:create]
```

Comment:
Good call on nesting this one route in here -- you're right, votes are inherently tied to works!